ci-operator/step-registry/ccm/conf/apply-feature-gate/ccm-conf-apply-feature-gate-commands.sh

```diff
@@ -37,23 +46,31 @@ export -f waitForCCMDeploymentCreation
 timeout --foreground 3m bash -c waitForCCMDeploymentCreation
 
 echo "$(date -u --rfc-3339=seconds) - Wait for operands to be ready"
-oc wait --all -n "${CCM_NAMESPACE}" --for=condition=Available=True deployment --timeout=3m
-
+err=$(oc wait --all -n "${CCM_NAMESPACE}" --for=condition=Available=True deployment --timeout=3m 2>&1 || true)
+# API server occasionally becomes unavailable, so some static wait is required for fault tolerance
+timedOutOrWait "$err" 1m
+# Repeat wait call for consistency
+oc wait --all -n "${CCM_NAMESPACE}" --for=condition=Available=True deployment --timeout=1m 
```

Comment:
does this give a total of 5 minutes for the ccm deployments to become active?

Reply:
Yes, at max, including a potential API server failure.

Reply:
Ok, now it does after https://github.com/openshift/release/pull/18138/commits/70c5f049eb2494ddbcdca2d0626ae3664ffad3a6 :) Thanks